human-readable-types/build.gradle

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-apply from: "$rootDir/gradle/publish-jar.gradle"
```

Comment:
Should we be removing `publish-jar.gradle`?

Reply:
huh, I thought I was doing that

Reply:
Oh I'm dumb, I forgot to commit that change as I thought it was leftover